pages/common/kdeconnect-cli.md

```diff
@@ -7,21 +7,21 @@
 
 `kdeconnect-cli --list-devices`
 
-- List all reachable devices:
+- List available (paired and reachable) devices:
 
 `kdeconnect-cli --list-available`
 
 - Request pairing with a specific device, specifying its ID:
 
 `kdeconnect-cli --pair --device {{device_id}}`
 
-- Request pairing with a specific device, specifying its name:
+- Ring a device, specifying its name:
 
-`kdeconnect-cli --pair --name {{device_name}}`
+`kdeconnect-cli --ring --name {{device_name}}`
```

Comment:
```suggestion
`kdeconnect-cli --name {{device_name}} --ring`
```

Reply:
I don't really think that's more intuitive, but it's fine too.